applications/launchpad/docker_rig/docker-compose.grafana.yml

```diff
@@ -0,0 +1,26 @@
+version: "3"
+
+networks:
+  loki:
+
+services:
```

Comment:
possibly loki and promtail will also need persistent volumes for whatever data/db/indexes they create

Reply:
This will be hooked into the launchpad container setup. We could create volumes for these, sure.